components/org.wso2.carbon.siddhi.editor.core/src/main/resources/web/commons/js/dialog/docker-export-dialog.js

```diff
@@ -51,6 +51,9 @@ define(['require', 'jquery', 'log', 'backbone', 'file_browser'],
 
                     var fileBrowser = new FileBrowser({container: treeContainer, application: self.app, fetchFiles: true,
                         showWorkspace: true, multiSelect: true});
+                    $(treeContainer).on('ready.jstree', function () {
+                        $(treeContainer).jstree("open_all");
```

Comment:
Use single quotes wherever possible instead of double quotes.

Reply:
And can't you access `$(treeContainer)` as `$(this)` within the function?

Reply:
added the changes